pkg/cli/dump_test.go

```diff
@@ -441,3 +441,75 @@ INSERT INTO t (i, j) VALUES
 		t.Fatalf("unexpected output: %s", out)
 	}
 }
+
+func TestDumpInterleaved(t *testing.T) {
```

Comment:
Can you use the testdata directory for these? I think it can handle all of this.

Reply:
https://github.com/cockroachdb/cockroach/tree/master/pkg/cli/testdata/dump is what I'm referring to here.